packages/bpk-component-link/index.js

```diff
@@ -18,7 +18,9 @@
 
 import BpkLink from './src/BpkLink';
 import BpkButtonLink from './src/BpkButtonLink';
-import themeAttributes from './src/themeAttributes';
+import themeAttributes, {
+  linkWhiteThemeAttributes,
+} from './src/themeAttributes';
 
 export default BpkLink;
-export { BpkButtonLink, themeAttributes };
+export { BpkButtonLink, themeAttributes, linkWhiteThemeAttributes };
```

Comment:
This still refers to these attributes as `linkWhiteThemeAttributes`.

Reply:
I'll update it now : )

Reply:
Updated. Can you have another look when you get time? : )